lib/versioner/external.go

```diff
@@ -81,3 +81,11 @@ func (v External) Archive(filePath string) error {
 	}
 	return errors.New("Versioner: file was not removed by external script")
 }
+
+func (v External) Rename(from, to string) error {
```

Comment:
We must give the external versioner a chance to act here. That's an API change, so it needs to go into v0.14 as we're releasing v0.13 tomorrow. As such we have some extra time to consider this, so no panic on driving this home just now.